cluster/gce/gci/configure-helper.sh

```diff
@@ -2104,6 +2104,12 @@ EOF
     prepare-kube-proxy-manifest-variables "$src_dir/kube-proxy/kube-proxy-ds.yaml"
     setup-addon-manifests "addons" "kube-proxy"
   fi
+  # Setup cluster monitoring using prometheus
```

Comment:
AFAIU we may still need to run standalone Heapster in addition to this - for users that use older versions of kubectl

Reply:
I will remove prometheus from cluster monitoring addons to allow deploy heapster separately.